radixdlt/babylon-wallet-android
# [ABW-3727] Refresh security problems on mnemonic deletion

## Description
- observe secure preferences size changes to trigger reload of entities with security problems


## How to test

1. Open wallet screen which is supposed to render security problems (if any)
2. Go to settings, remove and re-ad lock screen.
3. Go back to wallet - recovery required prompt do not show until you re-launch a wallet.
4. After the fix prompt should show even if wallet is in memory
